### ISSUE TITLE
add okc# to user groups list

### ISSUE THREAD

**Issue by amandaharlin** [techlahoma/techlahoma_lander#25]

**Comment by seejessicacode** [comment]
https://github.com/techlahoma/techlahoma/issues/58

**Comment by amandaharlin** [comment]
Don't have time to do this before the town hall. That's jesse and ny top priority rn

**Comment by seejessicacode** [comment]
Word. The main reason I'm going through all the GitHub issues is to collect
info that may come up in the Town Hall. If an overview of outstanding
issues isn't needed for the townhall then we'll just look over it at the
next board meeting.

**Comment by jagthedrummer** [comment]
If somebody can compile the info that we need to post for them I'd be happy to get it into the repo. The standard stuff we have for groups is something like this:

```ruby
{
    "name": "okc ruby",
    "website": "http://okcruby.org",
    "logo_url": "/images/logos/okcruby.png",
    "twitter": "okcrb",
    "location": "prototek",
    "meetupdate": "2nd Thursday",
    "email": "info@okcruby.org",
    "blurb": "OKC Ruby is focused on helping Oklahoma Rubyists to connect and share ideas. We hope you will join us."
  }
```

**Comment by seejessicacode** [comment]
Msg'd @matthewreily on Slack. Will see what I can find on my own also.

**Comment by seejessicacode** [comment]
{
    "name": "okc#",
    "website": "http://okcsharp.net/",
    "logo_url": "",
    "twitter": "okcsharp",
    "location": "prototek",
    "meetupdate": "1st Monday",
    "email": "matt@okcsharp.net",
    "blurb": "OKC# is a .Net centric usergroup located in Oklahoma City, Oklahoma. Oklahoma City is an emerging technology hub with hundreds of .Net developers working in the area."
  }

I stole the blurb from http://okcsharp.net/about/ and removed some redundancy.

Still need logo url.

**Comment by matthewreily** [comment]
Do you need a url or just a image file?

**Comment by seejessicacode** [comment]
I think we need the image file. Ruby example url is internal.

**Comment by matthewreily** [comment]
Will this work?
![okcsharp](https://cloud.githubusercontent.com/assets/8399844/15542812/e62c221e-2257-11e6-8fc0-f515de9b2ade.png)

**Comment by jagthedrummer** [comment]
Yep, that'll work!

**Comment by jagthedrummer** [comment]
Fixed in #27
